extensions/ql-vscode/src/remote-queries/shared/variant-analysis.ts

```diff
@@ -7,7 +7,8 @@ export interface VariantAnalysis {
   query: {
     name: string,
     filePath: string,
-    language: VariantAnalysisQueryLanguage
+    language: VariantAnalysisQueryLanguage,
+    queryText: string,
```

Comment:
Shall we call this `text` instead? We're inside the `query` object and we haven't prefixed other properties with that.